docker-compose.yml

```diff
@@ -4,3 +4,8 @@ services:
     image: postgres:11.5-alpine
     ports:
       - '127.0.0.1:5433:5432'
+    healthcheck:
+        test: ["CMD", "pg_isready", "-U", "postgres"]
+        interval: 0.1s
+        timeout: 1s
+        retries: 10
```

Comment:
Using Docker / Docker Compose's healthcheck thing allows each container's healthcheck command to be specified with the container in `docker-compose.yml`.

Also, it turns out Postgres has a builtin `pg_isready` for checking it.